lib/src/main/java/com/mapzen/pelias/http/Tls12OkHttpClientFactory.java

```diff
@@ -0,0 +1,64 @@
+package com.mapzen.pelias.http;
+
+import android.os.Build;
+import android.util.Log;
+
+import java.security.KeyStore;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import javax.net.ssl.SSLContext;
+import javax.net.ssl.TrustManager;
+import javax.net.ssl.TrustManagerFactory;
+import javax.net.ssl.X509TrustManager;
+
+import okhttp3.ConnectionSpec;
+import okhttp3.OkHttpClient;
+import okhttp3.TlsVersion;
+
+/**
+ * Enables TLS 1.2 support for pre-lollipop on {@link OkHttpClient.Builder} objects.
+ */
+public class Tls12OkHttpClientFactory {
+
+  /**
+   * Enables TLS 1.2 support for pre-lollipop on given {@link OkHttpClient.Builder}.
+   * @param client Client to enable TLS on.
+   * @return TLS enabled client.
+   */
+  public static OkHttpClient.Builder enableTls12OnPreLollipop(OkHttpClient.Builder client) {
+    if (Build.VERSION.SDK_INT >= 16 && Build.VERSION.SDK_INT < 22) {
```

Comment:
I think what I remember there might be something similar on some recent version of android also (7.0?). Though we have not updated that on tangram-es (https://github.com/tangrams/tangram-es/pull/1501). So will be good to document this and keep an eye for this in the future.